src/datastore/postgres-store.ts

```diff
@@ -816,7 +818,7 @@ export class PgDataStore extends (EventEmitter as { new (): DataStoreEventEmitte
       `
       INSERT INTO txs(
         ${TX_COLUMNS}
-      ) values($1, $2, $3, $4, $5, $6, $7, $8, $9, $10, $11, $12, $13, $14, $15, $16, $17, $18, $19, $20, $21, $22, $23, $24, $25, $26, $27)
+      ) values($1, $2, $3, $4, $5, $6, $7, $8, $9, $10, $11, $12, $13, $14, $15, $16, $17, $18, $19, $20, $21, $22, $23, $24, $25, $26, $27, $28)
```

Comment:
Is there no way to do this without having to manually type out a series of numbers? Seems like a hassle 😅

Reply:
You'd think the sql driver would support a spread operator or something (maybe it does and I haven't found it idk). 

At this point, probably makes sense to make a helper function that generates this string based off the array of input values.